part-time-schedule/week-5.md

```diff
@@ -0,0 +1,62 @@
+---
+description: Mastering Lightning and Node Management Workshop
+---
+
+# Week 5
+
+This week, we'll continue with "Mastering Lightning" book, holding Discussions in small groups as you answer discussion questions. We'll also be having workshops on Node Management.
+
+Also, we'll be working on more projects or expanding, refining or improving a project you have already started. 
+
+As usual, a web-reader-friendly hosted version of Mastering Lightning Book can be found at [lnbook.256k1.dev](https://lnbook.256k1.dev), which can be accessed for educational purposes using username and password `qala:lightning`.
+
+## Prerequisites
+
+* All the previous weeks' developer environments for both Bitcoin and Lightning Network.
+* All the previous chapters of Mastering Lightning Book.
+
+## Goals
+
+At the end of the week, we hope that you will:
+
+* Gain practical knowledge of how to manage a Mining Node
+* Understand Channel Operation and Payment Forwarding
+  * Managing HTLCs
+  * Committing HTLCs to Channel state
+  * Channel Balance
+* Understand Path Finding and Payment Delivery
+* Understand Lightning Payment Requests
+
+
+## Schedule
+
+Mastering Lightning reading and Node Management workshops are scheduled as follows:
+
+| Day       | Chapter/Workshop                                                                                                                                                                                                                      |
+| --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Monday    | [Chapter 9: Channel Operation and Payment Forwarding](https://github.com/lnbook/lnbook/blob/develop/09_channel_operation.asciidoc)                                                                  |
+| Tuesday   | Node Management Workshop                                      |
+| Wednesday | [Chapter 12: Pathfinding and Payment Delivery](https://github.com/lnbook/lnbook/blob/develop/12_path_finding.asciidoc)                                                                            |
+| Thursday  | Node Management Workshop  |
+| Friday    | [Chapter 15: Lightning Payment Requests](https://github.com/lnbook/lnbook/blob/develop/15_payment_requests.asciidoc)                                                                                        |
```

Comment:
This is twice the pace that we have been going so far, which is half the allocated amount of chapters from the full time seminar. I think this was a good pace? Would suggest for this week to do Ch9 on Mon/Wed and Ch12 pt1 on Fri? (this also affects `## Goals`)

Also suggest using Will's links, e.g. : https://lnbook.256k1.dev/#channel_operation